backend-status/Cargo.toml

```diff
@@ -0,0 +1,49 @@
+[package]
+name        = "orb-backend-status"
+version     = "0.0.0"
+description = "Systemd service that receives orb status and periodicallyprovides it to the Orb backend"
+authors     = ["Paul Quinn <paulquinn00@users.noreply.github.com>"]
+publish     = false
+
+edition      = { workspace = true }
+license      = { workspace = true }
+repository   = { workspace = true }
+rust-version = { workspace = true }
+
+[dependencies]
+chrono                  = { workspace = true, features = ["serde"] }
```

Comment:
what happened with the formatting? Was this from some autoformatter?

usually we just do edition.workspace = true, etc